dotdigital section updates for 2.4.2 release (#896)
magento/merchdocs: src/configuration/dotdigital/developer.md

|Order|Website|Sets the cron job interval to synchronize order data.|
|Catalog|Website|Sets the cron job interval to synchronize catalog data.|
|Review and Wishlist|Website|Sets the cron job interval to synchronize product review and wish list data.|

## System Alerts

![System Alerts]({% link images/images/dotdigital-dash-systemalerts.png %}){: .zoom}
_System Alerts_

|Field|[Scope]({% link configuration/scope.md %})|Description|
|--- |--- |--- |
|Enable System Messages|Global|Determines if in-app system messages are sent to admin users when errors that may affect the dotdigital extension are detected. Options: Yes / No|
|Enable Email Notifications|Global|Determines if system alert email notifications are sent to users with relevant permissions when errors that may affect the dotdigital extension are detected. Options: Yes / No|

> |Select Roles|Global|Specifies the user role(s) that will receive system alert email notifications. To select multiple roles, select the **Ctrl** key.|

Reviewer: ```suggestion
|Select Roles|Global|Specifies the user role(s) that receive system alert email notifications. To select multiple roles, select the **Ctrl** key.|
```

> |Alert Frequency|Website|Specifies both the email notification frequency and the time period for when the extension reports errors. Options: 1 Hour / 12 Hours / 24 Hours / 48 Hours / 72 Hours|

Reviewer: ```suggestion
|Alert Frequency|Website|Specifies both the email notification frequency and the time interval between error reports. Options: 1 Hour / 12 Hours / 24 Hours / 48 Hours / 72 Hours|
```

Author: @ccondra This one is a bit tricky. Alert Frequency 24hrs means:
- you get emails once every 24 hours
- only errors from the last 24 hours will be flagged

We wrestled with the phrasing on this one as well.

Reviewer: Yes, it is tricky!  I added your example to make it clear. Thanks!
